src/components/filterSelection/FilterSelection.js

```diff
@@ -69,7 +69,7 @@ const FilterSelection = ({ label, modificationText, applied, status, amountAppli
   return (
     <Box
       ref={ref}
-      className={cx(theme['select-control'])}
+      className={cx(theme['select-control'], status === Status.DEFAULT && theme['select-control--hovered'])}
```

Comment:
silly pp so feel free to ignore
`--hovered` threw me off, it's not actually being hovered yet so something like `--hoverable` seems more appropriate